docs/_static/references.bib

```diff
@@ -150,4 +150,28 @@ @book{church_murray_2018
   year={2018},
 }
 
+@article{MALISZEWSKI2012331,
+author = {Paul J. Maliszewski and Michael J. Kuby and Mark W. Horner},
+title = {A comparison of multi-objective spatial dispersion models for managing critical assets in urban areas},
+journal = {Computers, Environment and Urban Systems},
+doi = {https://doi.org/10.1016/j.compenvurbsys.2011.12.006},
```

Comment:
In DOIs you can let only the number. For example `doi = {10.1016/j.compenvurbsys.2011.12.006}`